rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -247,7 +247,7 @@
 
     private boolean shutedDown = false;
 
-    private Client caller = null;
+    private Client caller = localClient;
```

Comment:
Could you explain this please?

Reply:
Yes, I will write a comment there. Basically this is needed to avoid NPE during the recovery of the resource manager.